src/components/text/LargeParagraph.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+const LargeParagraph = ({ text }) => 
+	<main className="pa3 pa5-ns">
```

Comment:
Another consideration is if we should even include the container element at all, since its the same across all of them, may as well leave it out since we'll have 'container/layout' components eventually. Just a thought.